tests/component/test_watchdog.py

```diff
@@ -0,0 +1,81 @@
+from nidaqmx import Task
+from nidaqmx.constants import WatchdogAOExpirState, WatchdogCOExpirState
+from nidaqmx.system import Device
+from nidaqmx.system.watchdog import AOExpirationState, COExpirationState
+
+
+def test___call_cfg_watchdog_ao_expir_states___no_error(
+    generate_watchdog_task: Task, sim_9189_device: Device, sim_9263_device: Device
+):
+    watchdog_task = generate_watchdog_task(f"{sim_9189_device.name}", timeout=0.8)
+    expir_states = [
+        AOExpirationState(
+            physical_channel=sim_9263_device.ao_physical_chans[0].name,
+            expiration_state=0.0,
+            output_type=WatchdogAOExpirState.VOLTAGE,
+        ),
+        AOExpirationState(
+            physical_channel=sim_9263_device.ao_physical_chans[1].name,
+            expiration_state=0.0,
+            output_type=WatchdogAOExpirState.VOLTAGE,
+        ),
+    ]
+    watchdog_task.cfg_watchdog_ao_expir_states(expir_states)
+    watchdog_task.start()
+
+    assert not watchdog_task.expired
```

Comment:
arrange, act, assert
```suggestion
    ]

    watchdog_task.cfg_watchdog_ao_expir_states(expir_states)

    watchdog_task.start()
    assert not watchdog_task.expired
```

Reply:
Changed all